docs/source/en/_toctree.yml

```
@@ -55,6 +55,8 @@
 - sections:
   - local: optimization/fp16
     title: Memory and Speed
+  - local: optimization/torch2.0
+    title: Torch2.0 support.
```

```suggestion
    title: Torch2.0 support
```

A full stop in a page title looks very weird to me.